### DISCUSSION TITLE
Superfluous `<br />` tag in BitBucket on boarding PR comtent?

### DISCUSSION THREAD

**Discussion by jmartens** [renovatebot/renovate#29680] [Request Help] [answered]
### What would you like help with?

I think I found a bug

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

BitBucket cloud

### Please tell us more about your question or problem

I have been able to configure renovate to run against our BitBucket repositories and our private Azure Devops package streams for a testing repository, but I noticed a superfluous `<br />` tag in the onboarding pull request as can be seen in this screenshot highlighted by the orange arrow:

![image](https://github.com/renovatebot/renovate/assets/593327/a985efdc-7730-46a2-98ea-d96ef99db4f2)

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by jmartens** [comment]
I think it comes from https://github.com/renovatebot/renovate/blob/main/lib/workers/repository/onboarding/pr/pr-list.ts#L73

**Reply by rarkins** [reply, marked as answer]
I wondered how it got there and traced it all the way back to https://github.com/renovatebot/renovate/commit/1ada8b7913dacdf55ec14811e7dfa8fbba67c142#diff-278bee61badefed7ebce2908c1f18cede300a106b7572f1572cfb4df97e150bbR58

Fix PR: https://github.com/renovatebot/renovate/pull/29685
